### DISCUSSION TITLE
Markdown cropping (or a better way of doing code embeds)

### DISCUSSION THREAD

**Discussion by danielo515** [zsviczian/obsidian-excalidraw-plugin#1886] [Q&A]
Hello, and first thank you very much for this plugin.

I want to use it for exploring pieces of code and how they relate to each other. However, pasting each block of code individually is a bit tedious (I need to give it the language, a name and then it gets pasted). Ideally, I want to just have a big file (even, within the same code-block, so I don't have to do it several times) and then copy and paste it several times within excalidraw and adjust the "cropping" or "viewport" of each to the desired section.
Is this possible? can it be considered if it is not?
As far as I know, there are no good tools in the wild for this kind of workflow, so this will be a very unique feature of excalidraw.
Thanks in advance

**Comment by dangehub** [comment]
use `web embed` ? [[title#heading]] can embed part of a note


[how2_embed_part.webm](https://github.com/user-attachments/assets/9b7c00e8-4699-4dc3-8f7d-40695407bdee)

**Reply by danielo515** [reply]
That is what I've been doing so far, but it requires a separate code-block for each embed. I want to be able to just "focus" certain part of a big code block, not the whole code-block.
Am I making sense?
